buildlib/azure-pipelines-perf.yml

```diff
@@ -55,7 +90,7 @@ stages:
         pool:
           name: MLNX
           demands:
-          - ucx_perf
+          - ucx_perf_master
```

Comment:
why changed?

Reply:
This is a poor man's resource-locking mechanism: we only have a single agent labeled as 'master'. Another build will wait until it is free.
The 'ucx_perf' label is still used in matrix jobs, so we could run commands on both nodes.